SOURCES.toml

```
@@ -0,0 +1,630 @@
+[[resources]]  # Path: 7zip.png
```

```suggestion
#:schema https://datapackage.org/profiles/2.0/datapackage.json
[[resources]]  # Path: 7zip.png
```

This would be how you'd reference a schema for `taplo` https://taplo.tamasfe.dev/configuration/directives.html#the-schema-directive

I'm pretty sure we can't use a schema here though, since it represents an incomplete `datapackage`

---

>I'm pretty sure we can't use a schema here though, since it represents an incomplete datapackage

OK, My thought was that SOURCES.toml was not intended to stand alone as a datapackage, but would make a complete datapackage when consolidated with the json file generated by the script. I guess this could be hardcoded into the datapackage script, or just left out?